test/e2e/utils.go

```diff
@@ -191,6 +191,29 @@ func getJaegerOperatorImages(kubeclient kubernetes.Interface, namespace string)
 	return imageNamesMap, nil
 }
 
+func getJaegerOperatorNamespace() string {
+	if !usingOLM {
+		return namespace
+	}
+
+	var jaegerOperatorNamespace string
+	namespaces, err := fw.KubeClient.CoreV1().Namespaces().List(metav1.ListOptions{})
+	require.NoError(t, err)
+
+	for _, namespace := range namespaces.Items {
+		deployments, err := fw.KubeClient.AppsV1().Deployments(namespace.Name).List(metav1.ListOptions{})
+		require.NoError(t, err)
+		for _, deployment := range deployments.Items {
+			if deployment.Name == "jaeger-operator" {
+				return namespace.Name
+			}
+		}
+	}
+
+	logrus.Warn("Did not find a jaeger operator instance")
```

Comment:
Is enough to warn here? or should this fails? 

If is usingOLM and no namespace found this will return an empty string, and then other parts of the tests will fail.

Reply:
@rubenvp8510 Good point.  That should never really happen, but if it does something is really wrong, so failing is the correct behavior.